.github/workflows/publish-sdk.yml

```diff
@@ -0,0 +1,41 @@
+name: Tonomy SDK publish
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - development
+
+jobs:
+
+  publish-npm:
+    runs-on: ubuntu-20.04
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
+
+      
+      - name: 📦 Install
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.4.1
+          registry-url: https://registry.npmjs.org/
+
+      - name: "Automated Version Bump"
+        uses: "phips28/gh-action-bump-version@master"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          version-type:  'minor'
```

Comment:
version-type 'patch'

Reply:
actually, lets remove this line

Reply:
sure btw we can add changelog and rest of the things with it too @theblockstalk